python/composio/client/collections.py

```diff
@@ -516,7 +516,7 @@ def _handle_callback(
         try:
```

Comment:
Changing the log level from `info` to `error` is appropriate here given the context of an exception. Good change!